include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -1536,11 +1546,13 @@ struct __leaf_sort_kernel
 };
 
 // Please see the comment for __parallel_for_submitter for optional kernel name explanation
-template <typename _IdType, typename _LeafSortName, typename _GlobalSortName, typename _CopyBackName>
+template <typename _BackendTag, typename _IdType, typename _LeafSortName, typename _GlobalSortName,
```

Comment:
I put similar comment somewhere else. I don't understand why `_BackendTag` should be a part of class template, not the part of `operator()`

Reply:
Fixed

Reply:
Fixed